config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'list/index'
```

Comment:
I'd stick to the way of defining routes we already have, like in:
https://github.com/alphagov/govwifi-admin/pull/176/files#diff-21497849d8f00507c9c8dcaf6288b136R16